grails-app/views/occurrence/_map.gsp

```diff
@@ -399,7 +399,7 @@ a.colour-by-legend-toggle {
     }
 </r:script>
 
-<div class="hide">
+<div style="display: none;">
```

Comment:
So ``hide`` doesn't work? The map code hides elements by adding some class, not sure if it's ``hide``, but should probably test